woeip/apps/core/models.py

```diff
@@ -1,6 +1,12 @@
 from django.contrib.auth.models import AbstractUser
+from django.db import models
 
 
 class User(AbstractUser):
     class Meta:
         get_latest_by = 'date_joined'
+
+    email = models.CharField(max_length=256)
```

Comment:
`AbstractUser` already has an `email` field. This file does not need to be modified at this time.